If CC supports the "-print-multi-os-directory" option, use it to determine
where to check for the existence libcrypt.  If it doesn't, then we get /usr/lib.

--- ppp-2.4.2/pppd/Makefile.linux	2003-11-27 16:55:19.000000000 -0500
+++ ppp-2.4.2/pppd/Makefile.linux	2004-03-10 17:44:04.000000000 -0500
@@ -6,6 +6,7 @@
 # Default installation locations
 MANDIR = $(DESTDIR)/share/man/man8
 INCDIR = $(DESTDIR)/include
+LIBDIR = $(DESTDIR)/lib/$(shell gcc -print-multi-os-directory 2> /dev/null)
 
 TARGETS = pppd
 
@@ -33,7 +34,7 @@
 
 # CC = gcc
 #
-COPTS = -O2 -pipe -Wall -g
+COPTS = -Wall %(cflags)s -DLIBDIR=\""$(LIBDIR)"\"
 LIBS =
 
 # Uncomment the next 2 lines to include support for Microsoft's
 
